### ISSUE TITLE
The result of accuracy and k is strange when I change the batch size

### ISSUE THREAD

**Issue by irsLu** [BGU-CS-VIL/DeepDPM#33]
Question is the results trained on STL-10 are too low with different batch size.  
The only difference of environment is that the version of python is 3.8. 
And the accuracy is 0.793, and k is 10 when I use the default command (as shown in the first, second quote). 
But when I only change the batch size (e.g., 512), the result is very strange (as shown in the third, fourth quote), especially k = 3. So my question is what happen when I change the batch size?

>  **The default command :**  `
python DeepDPM.py --dataset stl10 --init_k 3 --dir pretrained_embeddings/MOCO/STL10 --NIW_prior_nu 514 --prior_sigma_scale 0.05  --seed 0 --use_labels_for_eva   --save_checkpoints True --exp_name 'test'`

> **The result of default command :** NMI: 0.74268, ARI: 0.6775, acc: 0.793, final K: 10

>  **The command of changing batch size :** `python DeepDPM.py --dataset stl10 --init_k 3 --dir pretrained_embeddings/MOCO/STL10 --NIW_prior_nu 514 --prior_sigma_scale 0.05  --seed 0 --use_labels_for_eva  --batch-size 512 --save_checkpoints True --exp_name 'test1'`

> **The result of the command of changing batch size :** NMI: 0.52621, ARI: 0.29056, acc: 0.2952, final K: 3

**Comment by irsLu** [comment]
It is my fault. When I change the learning rate of clustering net, the results are almost similar to the default setting.
